worlds/AutoSNIClient.py

```diff
@@ -0,0 +1,33 @@
+
+from __future__ import annotations
+from typing import ClassVar, Dict, Tuple, Any, Optional
+
+
+class AutoSNIClientRegister(type):
+    game_handlers: ClassVar[Dict[str, AutoSNIClientRegister]] = {}
+
+    def __new__(mcs, name: str, bases: Tuple[type, ...], dct: Dict[str, Any]) -> AutoSNIClientRegister:
+        # construct class
+        new_class = super().__new__(mcs, name, bases, dct)
+        if "game" in dct:
+            AutoSNIClientRegister.game_handlers[dct["game"]] = new_class()
+        return new_class
+
+    @staticmethod
+    async def get_handler(ctx) -> Optional[SNIClient]:
```

Comment:
This line is giving the linter error, for using `SNIClient` before it is declared. I noticed a similar thing is done with `WebWorld` in `AutoWorld.py`. I'm not sure what I need to do to make this pass like that other case does.

Reply:
addressed in https://github.com/PoryGoneDev/Pory_Archipelago/pull/5